Issue #13663: EJB Persistent Timer feature for Jakarta

tkburroughs opened the pull request:
Add enterpriseBeansPersistentTimer-4.0 and basic FAT

for #13663

tkburroughs commented:
#build

brideck commented on dev/com.ibm.websphere.appserver.features/visibility/public/enterpriseBeansPersistentTimer-4.0/resources/l10n/io.openliberty.enterpriseBeansPersistentTimer-4.0.properties:
```diff
@@ -0,0 +1,17 @@
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature enables the use of persistent timers in Enterprise JavaBeans.
```
Should we still be using Enterprise JavaBeans here in the description?

tkburroughs replied:
Nice catch; thanks.  I've updated all the translations to use Jakarta Enterprise Beans and also fixed the copyright.

tkburroughs commented:
Updated:
- corrected copyrights
- correctly used Jakarta Enterprise Beans
- added `WLP-Activation-Type: parallel` (all new features should use this)

tkburroughs commented:
#run-libby-bot

tkburroughs commented:
#build
